toxcore/tox.c

```diff
@@ -1595,6 +1605,9 @@ uint32_t tox_file_send(Tox *tox, uint32_t friend_number, uint32_t kind, uint64_t
     }
 
     /* can't happen */
+    LOGGER_FATAL(tox->m->log, "impossible return value: %ld", file_num);
+    assert(false);
```

Comment:
These will never be reached. LOGGER_FATAL already calls `abort()` unconditionally.